flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/batch/sql/join/ShuffledHashSemiAntiJoinTest.scala

```diff
@@ -164,22 +164,16 @@ class ShuffledHashSemiAntiJoinTest extends SemiAntiJoinTestBase {
 
   @Test
   override def testExistsWithCorrelated_LateralTableInSubQuery(): Unit = {
-    thrown.expect(classOf[TableException])
-    thrown.expectMessage("Cannot generate a valid execution plan for the given query")
     super.testExistsWithCorrelated_LateralTableInSubQuery()
   }
 
   @Test
   override def testInWithUncorrelated_LateralTableInSubQuery(): Unit = {
-    thrown.expect(classOf[TableException])
-    thrown.expectMessage("Cannot generate a valid execution plan for the given query")
     super.testInWithUncorrelated_LateralTableInSubQuery()
   }
 
   @Test
   override def testInWithCorrelated_LateralTableInSubQuery(): Unit = {
-    thrown.expect(classOf[TableException])
-    thrown.expectMessage("Cannot generate a valid execution plan for the given query")
     super.testInWithCorrelated_LateralTableInSubQuery()
```

Comment:
remove those three methods from this class, because the code is same with the super methods